core/chainlink.devspace.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Build image: Chainlink binary
+FROM golang:1.20-buster as buildgo
```

Comment:
buster is getting to be a decent bit old, may want to move to the latest stable which is bookworm https://www.debian.org/releases/

Reply:
Ignore me, buster is still in LTS